XmlSchemaClassGenerator.Console/Program.cs

```diff
@@ -170,7 +170,21 @@ static void Main(string[] args)
             generator.Generate(uris);
         }
 
-        static KeyValuePair<NamespaceKey, string> ParseNamespace(string nsArg, string namespacePrefix)
+        static void ShowHelp(OptionSet p)
+        {
+            System.Console.WriteLine("Usage: dotnet xscgen [OPTIONS]+ xsdFile...");
+            System.Console.WriteLine("Generate C# classes from XML Schema files.");
+            System.Console.WriteLine("Version " + typeof(Generator).Assembly.GetName().Version);
+            System.Console.WriteLine(@"xsdFiles may contain globs, e.g. ""content\{schema,xsd}\**\*.xsd"", and URLs.");
+            System.Console.WriteLine(@"Append - to option to disable it, e.g. --interface-.");
+            System.Console.WriteLine();
+            System.Console.WriteLine("Options:");
+            p.WriteOptionDescriptions(System.Console.Out);
+        }
+    }
+    public static class Utility
+    {
+        public static KeyValuePair<NamespaceKey, string> ParseNamespace(string nsArg, string namespacePrefix)
```

Comment:
Perhaps it's better to move this method into the `XmlSchemaClassGenerator` library project so the test project doesn't need a reference to the console project. This will probably also resolve the coverage issue.

Reply:
You are absolutely right, I did not see there is an existing class for utility purposes.
My apologies. I made a fix for this and will add a test for ParseNamespace method to improve the code coverage.
Thank you